substrate/primitives/core/src/paired_crypto.rs

```diff
@@ -71,6 +75,60 @@ pub mod ecdsa_bls377 {
 	impl super::CryptoType for Pair {
 		type Pair = Pair;
 	}
+
+	#[cfg(feature = "full_crypto")]
+	impl Pair {
+		/// Hashes the `message` with the specified `MsgHasher` and then signs it using ECDSA
```

Comment:
I'm confused. This was already integrated by: https://github.com/paritytech/polkadot-sdk/pull/2066

Reply:
Can you align to master first?

Reply:
Sorry, I think, I merged with the old PR when you said you are almost happy with it but I think you pushed few changes before you merge the last PR. I merged with master.